integration/testdata/fixtures/db/vulnerability.yaml

```diff
@@ -887,7 +887,7 @@
           V3Vector: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
       CweIDs:
       - CWE-502
-      Description: A deserialization of untrusted data vulnernerability exists in rails < 5.2.4.3, rails < 6.0.3.1 that can allow an attacker to unmarshal user-provided objects in MemCacheStore and RedisCacheStore potentially resulting in an RCE.
+      Description: A deserialization of untrusted data vulnerability exists in rails < 5.2.4.3, rails < 6.0.3.1 that can allow an attacker to unmarshal user-provided objects in MemCacheStore and RedisCacheStore potentially resulting in an RCE.
```

Comment:
These CVEs have errors in them. I'm happy to drop the files. If someone knows offhand how to get the CVEs fixed, that'd probably be better for the community overall.